recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,1449 @@
+                "libharu":           "libharu/[>=2.4.3]",
+                "kissfft":           "kissfft/[>=131.1.0]",
+                "lz4":               "lz4/[>=1.9.4]",
+
+                # TODO uncomment when no longer forcing ... "libpng":            "libpng/[>=1.6.39]",
+
+                "proj":              "proj/[>=9.1.1]",
+                "pugixml":           "pugixml/[>=1.13]",
+
+                # TODO uncomment when no longer forcing ... "sqlite3":           "sqlite3/[>=3.41.1]",
+
+                "utfcpp":            "utfcpp/[>=3.2.3]",
+                "xz_utils":          "xz_utils/[>=5.4.2]", # note: VTK calls this lzma
+                "zlib":              "zlib/[>=1.2.13]",
+                "TIFF":              "libtiff/[>=4.4.0]",
+                }
+
+        # NOTE: You may NOT be able to just adjust the version numbers in here, without
+        #   also adjusting the patch, as the versions are also mentioned in ThirdParty/*/CMakeLists.txt
+
+        if self.options.with_jpeg == "libjpeg":
+            parties["jpeg"] = "libjpeg/9e"
+        elif self.options.with_jpeg == "libjpeg-turbo":
+            parties["jpeg"] = "libjpeg-turbo/[>=2.1.5]"
+
+        if self._is_module_enabled([self.options.group_enable_StandAlone]):
+            # TODO uncomment when no longer forcing ... parties["hdf5"]    = "hdf5/[>=1.14.0]"
+
+            parties["theora"]  = "theora/[>=1.1.1]"
+            parties["ogg"]     = "ogg/[>=1.3.5]"
+            parties["netcdf"]  = "netcdf/[>=4.8.1]"
+
+            # TODO uncomment when no longer forcing ... parties["libxml2"] = "libxml2/[>=2.11.4]"
+
+            parties["cgns"]    = "cgns/[>=4.3.0]"
+
+        # unused dependency, mentioned in vtk but not actually used
+        # parties["zfp"]     = "zfp/[>=0.5.5]"
+
+        if self.options.build_all_modules:
+            parties["boost"]  = "boost/[>=1.82.0]"
+            parties["odbc"]   = "odbc/[>=2.3.11]"
+
+        if self._is_module_enabled([self.options.module_enable_nlohmannjson]):
+            parties["nlohmannjson"] = "nlohmann_json/[>=3]"
+
+        if self._is_module_enabled([self.options.module_enable_RenderingOpenVR]):
+            parties["openvr"] = "openvr/[>=1.16.8]"
+
+        if self._is_any_Qt_enabled:
+            if self.options.qt_version == "5":
+                parties["qt"] = "qt/[>=5.15.9]"
+            else:
+                parties["qt"] = "qt/[>=6.5.0]"
+
+        return parties
+
+
+    def requirements(self):
+        if self._is_module_enabled([self.options.group_enable_Rendering]):
+            self.requires("opengl/system")
+            if self.settings.os in ["Linux", "FreeBSD"]:
+                self.requires("xorg/system")
+        for pack in self._third_party().values():
+            self.requires(pack)
+
+        # TODO unhack this, fix up QT recipe instead?  Avoid openssl conflicts
+        # force versions to avoid build conflicts
+        # self.requires("openssl/[>=1.1 <4]")
```

Comment:
```suggestion
```

Already commented out, can be removed. OpenSSL is used as a version range in both Qt 5 and 6 recipes, so no conflict

Reply:
Yes, but there now a bunch of other `self.requires("something/ver", force=True)` that I had to add just to get it to build, thanks to dependency version clashes.  What is the correct way to deal with those problems?
Do I have to bump requirements to those dependencies (and then remove the hacks in this recipe) ?